drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/reports/index.xlsx.axlsx

```diff
@@ -23,11 +23,27 @@ wb.add_worksheet(name: 'User Permissions') do |sheet|
       user.id,
       user.overall_status(current_user).join('; '),
       user.agency_name,
-      user.last_sign_in_at&.to_time&.to_fs(:db),
+      user.last_warehouse_login&.to_time&.to_fs(:db),
       user.legacy_roles.map(&:name).join(', '),
       user.access_groups.filter(&:general?).map(&:name).join(', '),
       *@group_associations.keys.map { |ga| associated_items(user, ga)[:names].join(', ') }
     ]
     sheet.add_row(row, style: data_cell)
   end
 end
+
+if HmisEnforcement.hmis_enabled?
```

Comment:
Also check if this user as an `Hmis::User` would have permission `can_audit_users`. Weird to check an HMIS permission for a Warehouse report, but seems correct?